src/Client.java

```diff
@@ -21,5 +22,141 @@ public static void main(String args[]) throws Exception{
+        option = scanner.nextInt();
+        switch (option) {
+            case 1: {
+                processOpt1(stub);
+                break;
+            }
+            case 2: {
+                processOpt2(stub);
+                break;
+            }
+            case 3: {
+                System.out.println(processOpt3(stub));
+                break;
+            }
+            case 4: {
+                processOpt4(stub);
+                break;
+            }
+            case 5: {
+                System.out.println("GoodBye.");
+                break;
+            }
+        }
+        if (option == 5) return false;
+        else return true;
+    }
+
+    public void processOpt1(CenterServer stub) throws RemoteException {
+        Scanner scanner = new Scanner(System.in);
+        TeacherRecord teacher = new TeacherRecord();
+        System.out.println("Please input teacher's first name:");
+        teacher.setFirstName(scanner.nextLine().trim());
+        System.out.println("Please input teacher's last name:");
+        teacher.setLastName(scanner.nextLine().trim());
+        System.out.println("Please input teacher's address");
+        teacher.setAddress(scanner.nextLine().trim());
+        System.out.println("Please input your teacher's specialization:");
+        teacher.addSpecialiazation(scanner.nextLine().trim());
+        System.out.println("Please input teacher's location:");
+        teacher.setLocation(scanner.nextLine().trim());
+        System.out.println("Please input teacher's phone:");
+        teacher.setPhone(scanner.nextInt());
+        stub.createTRecord(teacher.getFirstName(), teacher.getLastName(), teacher.getAddress(), teacher.getPhone(), teacher.getSpecialiazation(), teacher.getLocation());
+    }
+
+    public void processOpt2(CenterServer stub) throws RemoteException {
+        Scanner scanner = new Scanner(System.in);
+        StudentRecord student = new StudentRecord();
+        System.out.println("Please input student's first name:");
+        student.setFirstName(scanner.nextLine().trim());
+        System.out.println("Please input student's last name:");
+        student.setLastName(scanner.nextLine().trim());
+        System.out.println("Please input student's status:");
+        student.setStatus(scanner.nextLine().trim());
+        System.out.println("Please input your student's statusDate:");
+        student.setStatusDate(scanner.nextLine().trim());
+        System.out.println("Please input student's courses(split with space):");
+        for (String s :
+                scanner.nextLine().split(" ")) {
+            student.getCoursesRegistered().add(s);
+        }
+        stub.createSRecord(student.getFirstName(), student.getLastName(), student.getCoursesRegistered(), student.getStatus(), student.getStatusDate());
+    }
+
+    public int processOpt3(CenterServer stub) {
+        //TODO: We should let this function return int and remove the argument "managerId".
```

Comment:
Why? 
expected format should contain server name and number of records, not 1 int,
so I suggest MTL:1 LVL:2 DDO:3... as the way to go format

Reply:
Make sense. I will remove it